packages/server/src/api/controllers/application.ts

```diff
@@ -154,11 +160,34 @@ async function createInstance(template: any) {
   } else {
     // create the users table
     await db.put(USERS_TABLE_SCHEMA)
+
+    if (includeSampleData) {
+      // create ootb stock db
+      await addDefaultTables(db)
+    }
   }
 
   return { _id: appId }
 }
 
+const addDefaultTables = async (db: any) => {
+  const defaultDbDocs = buildDefaultDocs()
+
+  // add in the default db data docs - tables, datasource, rows and links
+  await db.bulkDocs([...defaultDbDocs])
+
+  // Sync Quotas with the imported row count.
```

Comment:
This isn't necessary - we should just give them these rows for free and count any others they add